tox.ini

```diff
@@ -27,6 +27,7 @@ commands =
     # used by parser. This triggers flake8 error. Let's ignore this for now.
     flake8 --ignore F401 .
     py.test tests
+    py.test --doctest-modules sdf_timing
```

Comment:
I think you can add a flag to py.test to find doctests?

Reply:
I think that's what I've done with `--doctest-modules`. I may be able to combine the two lines and run at the top level. Is that what you mean?

Reply:
I was thinking it was part of https://docs.pytest.org/en/latest/goodpractices.html

Maybe it is just `py.test --doctest-modules` without any arguments?

Reply:
FYI I would just log a github issue to investigate and merge this as is...